data/dfndata/npc/mounts.dfn

```diff
@@ -810,4 +786,23 @@ NAME=an ethereal tiger
 ID=0x04e7
 SKIN=0x4001
 ORIGIN=hs
+}
+
+//hiryu - Base NPC with shared DFN tags
+[base_hiryu]
+{
+GET=base_mounts
+NAME=#//hiryu
+ID=0x0f3
+ORIGIN=se
+}
+
+[hiryu]
+{
+GETSE=hiryu_se
```

Comment:
Add one of these for each expansion, so adding NPC with wrong expansion selected doesn't break the server. We can update where these point at when we have definitions for different eras:

GETUO=hiryu_se
GETT2A=hiryu_se
GETUOR=hiryu_se
GETTD=hiryu_se
GETLBR=hiryu_se
GETAOS=hiryu_se
GETSE=hiryu_se
GETML=hiryu_se
GETSA=hiryu_se
GETHS=hiryu_se
GETTOL=hiryu_se